config/locales/de.yml

```diff
@@ -466,6 +468,7 @@ de:
       default_linter_feedback: Sehr gut. Der Linter hat nichts mehr zu beanstanden.
       error_messages: Fehlermeldungen
       existing_programming_group: Sie arbeiten gerade an der Übung mit dem Titel '%{exercise}' als Teil einer Programmiergruppe. Bitte schließen Sie Ihre Arbeit dort ab, indem Sie Ihren Code bewerten und abgeben, bevor Sie mit der Bearbeitung dieser Übung beginnen.
+      external_data_policy: Durch Nutzung des Videochats stimmen Sie der <a href=%{url}>externen Datenschutzerklärung</a> zu.
```

Comment:
The link is not clickable.

Reply:
You might need to use two == in the view.